lib/rubocop/config_loader.rb

```diff
@@ -245,10 +245,16 @@ def yaml_safe_load(yaml_code, filename)
         raise
       end
 
-      def yaml_safe_load!(yaml_code, filename)
-        YAML.safe_load(
-          yaml_code, permitted_classes: [Regexp, Symbol], aliases: true, filename: filename
-        )
+      def yaml_safe_load!(yaml_tree)
```

Comment:
I've got a slight concern that the name is pretty similar to `yaml_safe_load`, but the signature now is different. Maybe it should just be a method named differently?

Reply:
Yes, I suppose I should have renamed it. It's a private method, so no need to keep the old one around. New name is `yaml_tree_to_hash`, I guess? I did want to do some refactor to `YAMLDuplicationChecker`, and while it is marked private api, extensions use it to lint their own config file through the `project_spec` so I left that alone.